redesign-project/src/index.html

```diff
@@ -0,0 +1,67 @@
+---
+layout: layouts/default.njk
+title: Emma's Tea Spot
+---
+
+    {% include "_header.html" %}
+
+  <img src="https://lh3.googleusercontent.com/hrINEylFQ9GAW_WOJNL0CnnS-X9r8mBvemQsA1i79jzu2jGOtCy1SggNSjMG3rXaOc9tcT5RguefLjNLgsT1_6K7Ea98YyD7p1mDl7i9Z5Ti7Jzt6Kec1hKcXsFpA3IlKh2AmxX97d1_8dm9atd6RfamurWMW1Y41eWGPODkni5CbFYrTy5gM4TEFDuFNrl2YIxfICglgY7v9AgPQo5-Ys6Oyspl1662q9FP4U7v7CdLab5s-ivL8vhwz3URf6MKcspsnyNizA4tQUBajTJKneSaiV8XKaApDi7Gxh2qAM53WB8xV93GlTJ8ldCjeYx5P41yIswbUHVbIWogzY-1LYzKCHixpA5A3n8nIbZoLHbd071VyFZvjfdEClSKEUCGGib-_WaTheJvKM9-6CIG6-vtJBQya4oDK0xATSTvrFxQmtTsG0VmJta_hNw3V_VMeVq0TNwsn2EFLxYhf-GObboKpuTsNb6Lva7BjQBA4E12iqDvE66CHEB76zll0QtW4eU1CdvBws8USji28PNwebSjor9Af6oH0m14-u7z6-a4vaSaG-vB8vQwo6LH_133tT_Dukhn3NUFg442hQvQOHnOEX6tcMMh0OodNuhduTks_vJv-Iwcx5Pd2Mv5KMENi13uNAPTnSYcR0RQbsVz5PT9XouhBjW30iQqZtXPXM8E4Kn9j8jEiaLVQh3vfpd0lhqw3RL_GcfSt3ku=w882-h1344-no"
+    alt=" " class="background">
+
+    <img src="https://lh3.googleusercontent.com/BrkYF7ymKQaMQHOo8chYsPzUVzu1hHymNJpLfrXHKnL0y--NFu39TYjBucjWHnnFIkuA-6Htc4mEoSTrqPa48_B7t84GEs8mZjXzAzjPizGa9HK2A-0R7An6_zJDDkdxMGJV9yYXz1W6T4Ndd38JkqlPegUuF9O1D8p4YIxyU97MRrZIckkP70BOp6aejrJHDK1P1qKZFPQGYxZiaFJbRLPGf0bfGXA9Qj--YY4PyvtrT4k3bJcIXR0MyRUtARNYoUEmgISBeTcr6JUh_HqBj-agO-g_K3oyMM1kJad-e1SKtJPIfE4UiAofbPXSFtdg7F2l-tNX6MW0BmL5O_UtZtQwbKLV-Pw9tDvTqNTxbEpAQQ8qRVt-yEssZx44kx4eYVkieu_p7iCpBdSYGAOCx6kUnfbHh93yYDDY-0QhsSTg0XGf69jnjezV0ke8m7XwN0Ru7sKacRocLfftllJauZUXu1U7bUqAqUj_tHATfeEtRYSvR8flTQdAwCTA-iR7zdQftFtboEwIU-mzfNVXCzHbVXWLby0qYMhE2lqT2IXwjvpUQ7k_YktBd_gZKgSQNq2_U9bPkLCTiu2cM5BDxBxccMO6TGvFkxAIyX5n1cgb-z-jjYD4YOAXggSWgVTwCklztIfbG0htXGeWjqPeb43laoFZ0EncC4jUNKE-qRpK5Q9LVfAGy9dwibcds70LDS_MtRL4JSrra6GlmA=w2492-h1304-no"
+    alt=" " class="desktop_background">
+
+    <div class="rectangle"></div>
+
+    <div class="rectangle_large"></div>
+
+    <img src="https://static1.squarespace.com/static/59cee597bebafbd725f0d1f8/t/59cee7cad2b85766102d70fa/1540697069191/?format=1500w"
+        alt="Emma's Tea Spot: A Proper British Experience" class="header">
+
+
+    <ul>
+        <li>menu</li>
+        <li>reservations</li>
+        <li>events</li>
+        <li>shop</li>
+        <li>subscribe</li>
+    </ul>
+
+ <h1>Upcoming Events</h1>
```

Comment:
Currently, you're staggering your headings -- but is "Hours" really a subheading of "Upcoming Events"? If not, they should be at the same level -- most likely bet is that they should all be `h2`s.